config/webpack.common.config.js

```diff
@@ -19,4 +27,17 @@ module.exports = {
     },
     extensions: ['.js', '.jsx'],
   },
+  plugins: [
+    new webpack.DefinePlugin({
```

Comment:
Expose these variables as global variables usable by consuming MFEs and JS libraries.